plugins/GSdx/GSSetting.cpp

```diff
@@ -33,11 +33,16 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_FILTER:
 			return "Control the texture filtering of the emulation.\n\n"
 				"Nearest:\nAlways disable interpolation, rendering will be blocky.\n\n"
+				"Bilinear Forced/Upscale:\nAlways enable interpolation except for sprite. Rendering is smoother but it could generate few glitches.\n\n"
 				"Bilinear Forced:\nAlways enable interpolation. Rendering is smoother but it could generate some glitches.\n\n"
-				"Bilinear PS2:\nUse same mode as the PS2. It is the more accurate option.\n\n"
+				"Bilinear PS2:\nUse same mode as the PS2. It is the more accurate option.\n\n";
+#ifdef __unix__
+		case IDC_TRI_FILTER:
+			return "Control the texture tri-filtering of the emulation.\n\n"
+				"None:\nNo extra trilinear filtering.\n\n"
 				"Trilinear:\nUse OpenGL trilinear interpolation when PS2 uses mipmaps.\n\n"
-				"Trilinear Forced Bilinear:\nSame as above but always enable bilinear interpolation.\n\n"
 				"Trilinear Ultra:\nAlways enable full trilinear interpolation. Warning Slow!\n\n";
```

Comment:
It's called `Trilinear Forced` in code, but it's still `Trilinear Ultra` here. Either one needs to change. :)